deployment/gcp/infrastructure/production/provider.tf

```diff
@@ -0,0 +1,5 @@
+provider "google" {
+  project = var.gcp_project_id
+  region  = var.gcp_location
```

Comment:
Since "region" is how it is described by GCP and what TF is expecting, I think renaming the variable `gcp_location` to `gcp_region` makes sense.